engine/core/src/main/java/org/datacleaner/configuration/RemoteServerConfigurationImpl.java

```diff
@@ -56,5 +87,103 @@ public RemoteServerData getServerConfig(String serverName) {
         return null;
     }
 
+    @Override
+    public RemoteServerState getActualState(String remoteServerName) {
+        if (serverStatusTask == null) {
+            scheduledTask();
+        }
+        return actualStateMap.get(remoteServerName);
+    }
+
+    @Override
+    public void addListener(RemoteServerStateListener listener) {
+        if (serverStatusTask == null) {
+            scheduledTask();
+        }
+        listeners.add(listener);
+    }
+
+    @Override
+    public void removeListener(RemoteServerStateListener listener) {
+        listeners.remove(listener);
+    }
+
+    private synchronized void scheduledTask() {
```

Comment:
This seems to schdule a task, so it's scheduleTask not scheduledTask. And it would again be nice with a little more precise name, i.e. scheduleServerStatusTask. Not super important for private methods, but it still makes the code a little more readable, and everything helps :).